docs/resources/monitor.md

```diff
@@ -41,176 +44,73 @@ resource "datadog_monitor" "foo" {
-
-    We recommend at least 2x the monitor timeframe for metric alerts or 2 minutes for service checks.
-
--   `renotify_interval` (Optional) The number of minutes after the last notification before a monitor will re-notify
-
-    on the current status. It will only re-notify if it's not resolved.
-
--   `notify_audit` (Optional) A boolean indicating whether tagged users will be notified on changes to this monitor.
-
-    Defaults to false.
-
--   `timeout_h` (Optional) The number of hours of the monitor not reporting data before it will automatically resolve
-
-    from a triggered state. Defaults to false.
-
--   `include_tags` (Optional) A boolean indicating whether notifications from this monitor automatically insert its triggering tags into the title. Defaults to true.
--   `enable_logs_sample` (Optional) A boolean indicating whether or not to include a list of log values which triggered the alert. Defaults to false. This is only used by log monitors.
-
-    triggering tags into the title. Defaults to true.
-
--   `require_full_window` (Optional) A boolean indicating whether this monitor needs a full window of data before it's evaluated.
-
-    We highly recommend you set this to False for sparse metrics, otherwise some evaluations will be skipped. Default: True for "on average", "at all times" and "in total" aggregation. False otherwise.
-
--   `locked` (Optional) A boolean indicating whether changes to to this monitor should be restricted to the creator or admins. Defaults to False.
--   `tags` (Optional) A list of tags to associate with your monitor. This can help you categorize and filter monitors in the manage monitors page of the UI. Note: it's not currently possible to filter by these tags when querying via the API
--   `force_delete` (Optional) A boolean indicating whether this monitor can be deleted even if it’s referenced by other resources (e.g. SLO, composite monitor).
--   `monitor_threshold_windows` (Optional) A list of one element containing `recovery_window` and `trigger_window` values, e.g. `last_15m` . Can only be used for, and are required for, anomaly monitors.
-    -   `recovery_window` describes how long an anomalous metric must be normal before the alert recovers.
-    -   `trigger_window` describes how long a metric must be anomalous before an alert triggers.
--   `threshold_windows` (Deprecated, Optional) A dictionary defining the threshold windows for an anomaly monitor. Use `monitor_threshold_windows` instead.
--   `validate` (Optional) If set to false, skip the validation call done during `plan` .
--   `priority` (Optional) Integer from 1 (high) to 5 (low) indicating alert severity.
--   `silenced` (Optional) Each scope will be muted until the given POSIX timestamp or forever if the value is 0. Use `-1` if you want to unmute the scope. **Deprecated** The `silenced` parameter is being deprecated in favor of the downtime resource. This will be removed in the next major version of the Terraform Provider.
```

Comment:
Since this is deprecated, I think its OK we're not generating this section anymore ^